composer.json

```diff
@@ -24,6 +24,7 @@
 		"php": ">=7.1",
 		"bjeavons/zxcvbn-php": "^1.1.0",
 		"altis/browser-security": "^1.1.1",
+		"humanmade/disable-accounts": "~0.2.0",
```

Comment:
This could also be set to `^0.2.0` - I learned recently for the 0 major version even minor version changes are considered to be breaking so are not applied. Fine as is here but thought you'd find it interesting if you weren't already aware!

Reply:
Nice one, don't recall that, but I still get myself confused with Composer vs NPM SemVer:

- npm: `"~1.3"` = `>=1.3.0` `<1.4.0`
- composer: `"~1.3"` = `>=1.3.0` `<2.0.0`